### PR TITLE
Fixed #29630 -- Fixed crash of sliced queries with multiple columns with the same name on Oracle 12.1.

### PR DESCRIPTION
**Pull request by felixxm**
Ticket [29630](https://code.djangoproject.com/ticket/29630).

Regression in 0899d583bdb140910698d00d17f5f1abc8774b07.

### CODE REVIEW

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by jtiai** [comment]
Tested with Oracle 12.1 and tests do pass.

**Comment by felixxm** [comment]
@jtiai Many thanks for confirmation :rocket:

File: django/db/backends/oracle/features.py [hunk 1]
```diff
@@ -55,3 +56,13 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_over_clause = True
     supports_ignore_conflicts = False
     max_query_params = 2**16 - 1
+
+    @cached_property
+    def has_fetch_offset_support(self):
+        if tuple(int(x) for x in self.connection.oracle_full_version.split('.')) < (12, 2):
```
[the remark this answers is not in the document]
Comment (felixxm) [reply django/db/backends/oracle/features.py]: Reworked in #10442.

**Comment by felixxm** [comment]
@timgraham Thanks for the review. I updated PR.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Pull request merged**